checks.d/elastic.py

```diff
@@ -322,16 +322,24 @@ def check(self, instance):
         health_url, nodes_url, stats_url, pshard_stats_url, pending_tasks_url, stats_metrics, \
             pshard_stats_metrics = self._define_params(version, config.cluster_stats)
 
+        # Load stats data.
+        # This must happen before other URL processing as the cluster name
+        # is retreived here, and added to the tag list.
+
+        stats_url = urlparse.urljoin(config.url, stats_url)
+        stats_data = self._get_data(stats_url, config)
+        if stats_data['cluster_name']:
+            # retreive the cluster name from the data, and append it to the
+            # master tag list.
+            config.tags.append("cluster_name:{}".format(stats_data['cluster_name']))
```

Comment:
My 2 cents: worked a bit on the integration, and to me it makes sense to add it to `service_check_tags` too.